libs/cde-visualization/src/webcomponent-example.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>CDE Visualization</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <script>
+    window.addEventListener('DOMContentLoaded', () => {
+      const ftu = document.getElementById('cde');
+      const nodeClick = (nodeEntry) => {
+        console.log('node click', nodeEntry);
+      };
+      const nodeHover = (nodeEntry) => {
+        console.log('node hover', nodeEntry);
+      };
+      ftu.addEventListener('nodeClicked', nodeClick);
+      ftu.addEventListener('nodeHovered', nodeHover);
+    });
+  </script>
+  <body>
+    <div class="wc-content">
+      <cde-visualization-root
```

Comment:
Please provide actual values for these attributes. Nodes, edges, color-map, and metadata all accept urls to files